File: tests/mpas_intel_libs_check/test_files/pnetcdf.c
#include <pnetcdf.h>
#include <stddef.h>
#include "mpi.h"

int main(void) {

	int ncid, ierr;

    MPI_Init(NULL, NULL);
	ierr = ncmpi_create(MPI_COMM_WORLD, "blah.nc", 
										 NC_64BIT_OFFSET, 
										 MPI_INFO_NULL, 
										 &ncid);
	
	ierr = ncmpi_close(ncid);
	MPI_Finalize();

	return 0;
}

File: tests/mpas_pgi_libs_check/test_files/pnetcdf.f90
program pnetcdf_test

   use mpi
   use pnetcdf

   integer :: ierr, ncid

   call MPI_Init(ierr)

   ierr = nf90mpi_create(MPI_COMM_WORLD, 'pnetcdfBlah.nc', NF90_CLOBBER, MPI_INFO_NULL, ncid)
   ierr = nf90mpi_close(ncid)

   call MPI_Finalize(ierr)

   stop 

end program pnetcdf_test
